src/Tizen.NUI/src/public/XamlBinding/BindableObject.cs

```diff
@@ -310,7 +310,7 @@ internal void InternalSetValue(BindableProperty property, object value)
                 }
 
                 object oldvalue = null;
-                BindablePropertyContext context = GetOrCreateContext(property);
+                BindablePropertyContext context = GetOrCreateContext(property, true, true);
```

Comment:
이 함수(`InternalSetValue`)가 아마 SRCN에서 Binding된 케이스와 그렇지 않은 케이스를 구분하여 동작하기 위해 분기를 위해 별도로 작성한 함수로 알고 있습니다.

이 함수 내에서 아래와 같이 Context를 만들지 않는 방법으로 수정한 뒤 테스트 해보시면 어떨까요? (정상동작확인 + 성능 측정)
```suggestion
                BindablePropertyContext context = GetContext(property);
```

Reply:
안녕하세요. 제시해주신 방법도 흥미로워서 테스트 해봤습니다.

(아직은 Debug 모드 결과이긴 하지만 참고용으로만 보시면 될 것 같습니다.)

|기존|Context Create|Skip Context Create|
|---|---|---|
|7.0403462962962955 ms|6.185950000000002 ms|5.563459259259258 ms|

Context Create 를 하지 않는 것 만으로도 상당한 성능 개선이 되는 것을 확인했습니다.

다만 이렇게 수정한 경우, Xaml pasing 을 하는 NUITizenGallery 앱이 정상적으로 실행이 안되는 것을 확인하였는데요, 이는 추가적인 디버깅이 필요할 것 같습니다.

감사합니다.